compiler/rustc_target/src/callconv/x86.rs

```diff
@@ -223,18 +224,19 @@ where
     // registers will quiet signalling NaNs. Also avoid using SSE registers since they
     // are not always available (depending on target features).
     if !fn_abi.ret.is_ignore()
-        // Intrinsics themselves are not actual "real" functions, so theres no need to change their ABIs.
-        && abi != SpecAbi::RustIntrinsic
+        // Intrinsics themselves rustc_abi::functions, so theres no need to change their ABIs.
```

Comment:
Search and replace may have changed this comment unexpectedly

Reply:
Thank you~